modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/ClusterMetricsImpl.java

```diff
@@ -181,6 +207,7 @@ public ClusterMetricsImpl(GridKernalContext ctx, long nodeStartTime) {
         peakThreadCnt = mreg.findMetric(PEAK_THREAD_CNT);
         totalStartedThreadCnt = mreg.findMetric(TOTAL_STARTED_THREAD_CNT);
         daemonThreadCnt = mreg.findMetric(DAEMON_THREAD_CNT);
+        executedTasks = mreg.findMetric("TotalExecutedTasks");
```

Comment:
Let's make "TotalExecutedTasks" and other metric names constant in `GridIoManager`